### DISCUSSION TITLE
How do I theme GNOME's light and dark theme indepedently of eachother?

### DISCUSSION THREAD

**Discussion by sylvesterroos** [GradienceTeam/Gradience#322] [Q&A]
When I generate a theme using Monet it sets the theme for both the light and dark theme in GNOME. When I go into the settings and change the style from dark to light, it stays dark.

**Comment by 0xMRTT** [comment]
For some reason, gtk seems to ignore gtk-dark.css. The ability of selecting where the theme will be applied will come soon 🚀
